bin/make_view

```diff
@@ -51,34 +53,16 @@ CREATE OR REPLACE VIEW "Measure_Expenditures" AS
     "Ballot_Measure" as "Measure_Number",
     "Support_Or_Oppose" as "Sup_Opp_Cd",
     "Amount",
-    "Payee_NamL" as "Recipient_Or_Description"
+    "Payee_NamL" as "Recipient_Or_Description",
+    'E number' as "Form"
   FROM
     "E-Expenditure" expend
   JOIN oakland_committees committee
       ON expend."Filer_ID"::varchar = committee."Filer_ID"::varchar
   JOIN oakland_name_to_number ON "Measure_Name" = "Bal_Name"
   WHERE "Bal_Name" IS NULL
     AND "Ballot_Measure" IS NOT NULL
-  UNION ALL
-
-  -- Get 24hr report expenditures. Infers S/O information from "committees" tab
-  -- of spreadsheet.
-  SELECT
-    "497"."Filer_ID"::varchar,
-    "497"."Filer_NamL",
-    "election_name",
-    "Bal_Name",
-    "Measure_Number",
-    COALESCE(oakland_committees."Support_Or_Oppose", 'Unknown') as "Sup_Opp_Cd",
-    "Amount",
-    "Cmte_ID" || ' / ' || "Enty_NamL" as "Recipient_Or_Description"
-  FROM "497"
-  LEFT OUTER JOIN "oakland_name_to_number"
-    ON LOWER("Bal_Name") = LOWER("Measure_Name")
-  LEFT OUTER JOIN oakland_committees
-    ON oakland_committees."Filer_ID" = "497"."Cmte_ID"
-  WHERE "Bal_Name" IS NOT NULL
-  AND "Form_Type" = 'F497P2';
```

Comment:
Isn't this form used by more than just the major donors? Can we just `WHERE` them out of the results here?

Reply:
Yes it is used for all large contributions within the "late period", but this view is used to look at expenditures not contributions.